catalog/app/containers/Bucket/Qurator/Summary.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react'
+import * as Lab from '@material-ui/lab'
+
+import Chat from 'components/Chat'
+import type * as Model from 'model'
+import * as APIConnector from 'utils/APIConnector'
+import * as AWS from 'utils/AWS'
+import mkSearch from 'utils/mkSearch'
+
+const Loading = Symbol('Loading')
+
+type EsHit = { _source?: { content?: string } }
+
+type EsOutput = { hits?: { hits?: { _source?: { content?: string } }[] } } | null
+
+interface Hit {
+  value: EsHit | null
+}
+
+const FILE_PROMPT = (
+  content: string,
+) => `I will ask you questions about the file's content indexed by ElasticSearch. The ElasticSearch JSON output:
+
+${content}.
+
+Please summarize the content of this file intelligently and concisely. Focus on file's content, don't mention ElasticSearch if unnecessary.`
+
+const NO_CONTENT_PROMPT = (
+  json: string,
+) => `Please, tell me, that you can't answer questions about the content of the file. It is empty or not yet indexed. But you can tell about meta, indexed by ElasticSearch. This meta is:
```

Comment:
```suggestion
) => `Please respond that you can't answer questions about the content of the file, since it's empty or not yet indexed. However, you can tell about metadata indexed by ElasticSearch. This metadata is:
```